i18n-unused.config.js

```diff
@@ -0,0 +1,19 @@
+// Info: https://www.npmjs.com/package/i18n-unused
+
+module.exports = {
+  localesPath: 'packages/common/src/intl/locales',
+  srcPath: 'packages',
+  ignorePaths: [
```

Comment:
I don't think `ignorePaths` is necessary - in that it runs in the same amount of time without it. Tried with the glob syntax

```
  ignorePaths: ['**/node_modules'],
```

and then simply removed. Did you have an issue which required adding the prop in?

Reply:
Thanks, have used your suggestion below 👍🏼